.github/workflows/build.yml

```diff
@@ -8,14 +8,21 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
-
     steps:
       - uses: actions/checkout@v2
+
       - name: Set up JDK 11
         uses: actions/setup-java@v2
         with:
           java-version: '11'
           distribution: 'adopt'
           cache: 'maven'
-      - name: Build with Maven
-        run: mvn test
+
+      - name: Start Postgres
+        run: docker compose -f src/test/resources/docker-compose.yml up -d --wait
+
+      - name: Install test library to local Maven repo
+        run:  mvn install:install-file -Dfile=libs/jdbc-connector-for-apache-kafka-6.7.0-SNAPSHOT.jar -DgroupId=io.aiven -DartifactId=jdbc-connector-for-apache-kafka -Dversion=6.7.0-SNAPSHOT -Dpackaging=jar
+
+      - name: Run tests
+        run: mvn integration-test verify
```

Comment:
`verify` is needed so the build fails if any integration tests fails (by default, ITs do not fail builds).